### PR TITLE
ENH allow removal of profiles.csv; allow setting n_nodes in profile; make callosal bundles the default

### PR DESCRIPTION
**Pull request by 36000**
Whats remaining is to test some of these new features

### CODE REVIEW

**Comment by 36000** [comment]
@arokem this is ready for review and merge

File: AFQ/api/group.py [hunk 1]
```diff
@@ -886,6 +886,15 @@ def assemble_AFQ_browser(self, output_path=None, metadata=None,
...
                 "`pip install AFQ-Browser>=0.3`"))
             return
+        n_points_profile = self.export(
+            "n_points_profile", collapse=False)[
+                self.valid_sub_list[0]][
+                    self.valid_ses_list[0]]
+        if n_points_profile != 100:
+            self.logger.warning((
+                "AFQ Browser requires 100 points per tract profile, "
+                "so AFQ Browswer instance cannot be assembled."))
```
Comment (arokem) [inline AFQ/api/group.py:896]:
```suggestion
                "so AFQ Browser instance cannot be assembled."))
```

File: AFQ/api/group.py [hunk 2]
```diff
@@ -886,6 +886,15 @@ def assemble_AFQ_browser(self, output_path=None, metadata=None,
                 "`pip install pyAFQ[afqbrowser]` or "
                 "`pip install AFQ-Browser>=0.3`"))
             return
+        n_points_profile = self.export(
+            "n_points_profile", collapse=False)[
```
Comment (arokem) [inline AFQ/api/group.py:890]: Looks like this export call isn't working: https://github.com/yeatmanlab/pyAFQ/actions/runs/6880271899/job/18714094705?pr=1052#step:5:2321

**Review by arokem — COMMENTED** [review]
Looks like the documentation is not building cleanly with these changes. Looks like maybe `export("n_points_profile")` is not working?

**Pull request merged**